### ISSUE TITLE
renaming of syslog-ng-debun's man page

### ISSUE THREAD

**Issue by czanik** [balabit/syslog-ng#1609]
# syslog-ng

## Version of syslog-ng

git head as of 2017 July 19

# Issue

The man page of syslog-ng-debun is called syslog-debun. The man page should be also renamed to syslog-ng-debun to be consistent.

**Comment by presidento** [comment]
Please, create a ticket in the [documentation repository](https://github.com/balabit/syslog-ng-ose-guides). We just reuse the prepared xml documentation from there. We can update here only after that.

**Comment by presidento** [comment]
Update: as I see, it is already done there, I'll check it.

**Comment by presidento** [comment]
PR created, thx.
